packages/suite/src/components/suite/FormattedNumber/index.tsx

```diff
@@ -26,8 +26,8 @@ export default (props: Props) => {
         <FormattedNumber
             currency={props.currency}
             value={bn.toNumber()}
-            minimumFractionDigits={props.minimumFractionDigits || 2}
-            maximumFractionDigits={props.minimumFractionDigits || 2}
+            minimumFractionDigits={props.minimumFractionDigits ?? 2}
```

Comment:
nice! (??) :D

Reply:
yeah, I spent an ungodly amount of time wondering why passing '0' doesn't work